### PR TITLE
added a little info to notebook 2

### PR DESCRIPTION
**Pull request by IrisGlazeRTI**

### CODE REVIEW

**Review by jon-holt — APPROVED** [review]
Looks good, Iris!

File: content/Main_Notebooks/2 - Pandas Data Structures.ipynb [hunk 1]
```diff
@@ -2502,11 +2520,15 @@
     "\n",
     "However, right now we got only items in ID that were present in both dataframes.We can change this behavior with the keyword `how`. We have four simple options:\n",
     "- Inner: This is the default, where only items present in the matching column in both frames are included.\n",
-    "- Outer: All items are included.\n",
-    "- Left: All columns in the left dataframe (in the order you write them in `pd.merge()`) are kept, ones only present in the right dataframe are discarded.\n",
-    "- Right: The opposite of left.\n",
+    "- Outer: All rows from both dataframes are included.\n",
+    "- Left: All columns in the left dataframe (in the order you write them in `pd.merge()`) are kept. Ones only present in the right dataframe than don't match up with any rows in the left dataframe are discarded.\n",
```
Comment (michael-long88) [inline content/Main_Notebooks/2 - Pandas Data Structures.ipynb:3381]: > Ones only present in the right dataframe than don't match up 3

I think that "than" was supposed to be a "that"

File: content/Main_Notebooks/2 - Pandas Data Structures.ipynb [hunk 2]
```diff
@@ -2345,12 +2345,29 @@
...
-    "One thing that you do often need to do when you make dataframes is merge them together. We will need some very small dataframes to see this behavior in."
+    "\n",
+    "When you load your data from the workbench, there is a high likelihood that you will have to combine two pre-existing dataframes together to make a new dataframe. For instance, there may be a dataframe representing people, and another dataframe representing medication prescriptions. Before combining these dataframes, each row in the person dataframe will represent a different person. The medication prescriptions dataframe will have one column with a person_id, and other information in other columns with information on medications that someone is prescribed, such as the medication name, and prescription date. To use information from both of these tables at once, you will need to join or merge by the person_id column that exists in both dataframes.![image.png](attachment:91ccf989-69cc-47c0-9922-ff457bb3c503.png)![image.png](attachment:7d9c480e-0666-4637-8fab-af3976cb8400.png)![image.png](attachment:79d3f474-dca4-498e-b295-8c96670415a0.png)![image.png](attachment:290762c9-2835-45fe-b5a8-46c13611dba3.png)\n",
```
Comment (michael-long88) [inline content/Main_Notebooks/2 - Pandas Data Structures.ipynb:2887]: I'm not sure what happened, but it looks like image versions of the text was inserted here.

**Review by michael-long88 — CHANGES_REQUESTED** [review]

File: content/Main_Notebooks/2 - Pandas Data Structures.ipynb (hunk 1 above)
Comment (jon-holt) [reply content/Main_Notebooks/2 - Pandas Data Structures.ipynb:3381]: yup, you're right

File: content/Main_Notebooks/2 - Pandas Data Structures.ipynb (hunk 2 above)
Comment (jon-holt) [reply content/Main_Notebooks/2 - Pandas Data Structures.ipynb:2887]: Weird, I didn't notice this before. I'll remove the screenshots - it looks like they were intended to be temporary.

**Pull request merged**